build.gradle

```diff
@@ -55,7 +55,7 @@ intellij {
     type = 'IU'
     downloadSources = true
     updateSinceUntilBuild = false
-    plugins = ['JavaScript']
+    plugins = ['JavaScript','org.toml.lang:231.8109.1']
```

Comment:
The current dependency declared on JavaScript is optional. This block only affects the version of the IDE that's used for plugin dev. 
https://github.com/intellij-solidity/intellij-solidity/blob/cb7a60fa466f32157568740406794943414bb010/src/main/resources/META-INF/plugin.xml#L69-L70

From what I can see, if the TOML plugin is missing in one of the IDEs, it'll cause an exception in the code below that uses it.

Generally, since the only functionality that's actually required is parsing TOML files, including some tiny zero-dependency library rather than a plugin will avoid this issue as well as the risk of these interfaces changing in the next version.

Reply:
Thank you for your feedback
I see your point. What are you suggesting ? Writing a custom parser ?

Reply:
Adding a small 3rd party library that supports parsing TOML files can be the most reasonable option. From what I can tell, JacksonXML supports TOML files.

Reply:
I will try, thanks :pray: